### DISCUSSION TITLE
Home Screen icon for IOS not working

### DISCUSSION THREAD

**Discussion by sourfe** [muraliwebworld/push-notification-plugin-for-wordpress-websites#22] [Q&A]
When I add a site to homescreen on IOS I don't get any icon but works fine on other platforms. 
I can see in logs that it requests different apple-touch-icons with response code 400.

This used to work in earlier versions.

**Comment by muraliwebworld** [comment]
I will check and I will let you know by tomorrow. Since i am working on optimizing subscriptions of topic (wp_remote_post is taking more time to respond back while subscribing topics), I am replacing with other function which reduces more time. It is almost over, I will look into your issues tomorrow

**Comment by sourfe** [comment]
Did you have any chance to look into icon issue?

**Comment by muraliwebworld** [comment]
I will update you tomorrow on this

**Comment by muraliwebworld** [comment]
I have verified, it is working for me, i have shared below video link,  video captured from my ipad mini

**Video link**
https://drive.google.com/file/d/1yIQxoUun6WYcmp8EFGKnSZO423HsvEps/view?usp=sharing

Following are 3 images showing settings
https://drive.google.com/file/d/1DxwHdiS1TD3xBhDk64CO1nPt5Xd9IEsK/view?usp=sharing
https://drive.google.com/file/d/1m-NlXsIt5C5iQuD87_L3xoFRhfks3D9E/view?usp=sharing
https://drive.google.com/file/d/1h1XotUCW2Pu0l58kEofoPY5DMysQHVyH/view?usp=sharing

**Comment by sourfe** [comment]
Ok, thanks. 

I have wordpress in a subfolder but my site is accessed normally as  https://www.site.com/ . Can that be an issue?
This used to work with a version that is about 10 month old now.
The strange thing is that the icon that show up is not any icon that is referenced in the backend at all.

Do you have any suggestions how to troubleshoot? Database table etc?

**Comment by sourfe** [comment]
The correct icon is actually shown but only for as plit second and then replaced by the other icon.

**Comment by sourfe** [comment]
I finally found out the problem. 
When I try to add to homescreen, PNFPB tries all the below images and if one of them are found, it will replace the one I have specified. I made sure none of them was reachable and now it works.

Any idea why it is like this?

This is after I moved the files.
"GET /apple-touch-icon-precomposed.png HTTP/2" 404 50450"
 "GET /favicon.ico HTTP/2" 302 0"
 "GET /prodcms/wp-content/uploads/2023/12/cropped-favicon-32x32.png HTTP/2" 404 50498"
 "GET /prodcms/wp-content/uploads/2023/12/cropped-favicon-192x192.png HTTP/2" 404 50499"
 "GET /prodcms/wp-content/uploads/2023/12/cropped-favicon-180x180.png HTTP/2" 404 50499"
 "GET /apple-touch-icon.png HTTP/2" 404 50421"
 "GET /prodcms/wp-content/uploads/2023/12/cropped-favicon-32x32.png HTTP/2" 404 50498"
 "GET /prodcms/wp-content/uploads/2023/12/cropped-favicon-180x180.png HTTP/1.1" 404 50499"
 "GET /apple-touch-icon-120x120-precomposed.png HTTP/1.1" 404 50465"
 "GET /apple-touch-icon-120x120.png HTTP/1.1" 404 50436"
 "GET /apple-touch-icon.png HTTP/1.1" 404 50422"

**Comment by muraliwebworld** [comment]
I designed as per apple instructions, i will check again Apple pwa
documentation, I will update it in next release(next week)
